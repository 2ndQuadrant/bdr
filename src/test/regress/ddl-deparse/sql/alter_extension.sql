---
--- ALTER_EXTENSION
---

ALTER EXTENSION pg_stat_statements UPDATE;
