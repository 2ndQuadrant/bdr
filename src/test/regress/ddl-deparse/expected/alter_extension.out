---
--- ALTER_EXTENSION
---
ALTER EXTENSION pg_stat_statements UPDATE;
NOTICE:  version "1.2" of extension "pg_stat_statements" is already installed
